src/web/src/components/CatMenu/CatMenu.module.css

```diff
@@ -0,0 +1,5 @@
+.sideMenu {
+  padding-top: 3em;
```

Comment:
What is the use of `em`? are there alternatives?

Reply:
`em` units are proportional to the selected font size. In case a user has selected a bigger font the same `3em` value would result to a bigger actual distance. Alternatives are `px`, `pt`, `cm`, `in`.